server/src/bin/www.js

```diff
@@ -11,45 +11,28 @@ require('dotenv').config();
  */
 const http = require('http');
 const app = require('../app');
+const config = require('../config');
 
 /**
  * Get port from environment and store in Express.
  */
-const port = normalizePort(process.env.PORT) || '3001';
-app.set('port', port);
+const { port } = config.web;
+app.set('port', config.web.port);
```

Comment:
just use port here instead of config.web.port?

Reply:
Fixed & pushed